src/main/flight/interpolated_setpoint.c

```diff
@@ -102,11 +102,11 @@ FAST_CODE_NOINLINE float applyFfLimit(int axis, float value, float Kp, float cur
         DEBUG_SET(DEBUG_FF_LIMIT, 0, value);
     }
 
-    if (axis == FD_ROLL) {
+    if (axis == FD_PITCH) {
```

Comment:
This should be refactored into a switch to avoid having the same kind of duplication in the future.